src/components/Editor/Invitees/OrganizerNoEmailError.vue

```diff
@@ -26,7 +26,7 @@
 			@
 		</div>
 		<div class="editor-invitee-list-no-email-configured-message__caption">
-			{{ $t('calendar', 'Please configure your email-address in the personal settings. It is necessary to send out invitations and handle responses of your invitees.') }}
+			{{ $t('calendar', 'To send out invitations and handle responses, an email address is necessary.') }}
```

Comment:
I know @jancborchardt wanted this shorter, but I feel we should still say where to fill the email.
Is the following still too long?
> To send out invitations and handle responses, you need to configure your email address in your personal settings.

Reply:
> To send out invitations and handle responses, add your email address in personal settings.

And the "add your email address in personal settings" should be linked to the personal settings if possible.

Reply:
On it ...